frontend/views/pages/BypassUI.vue

```diff
@@ -0,0 +1,150 @@
+<template lang='pug'>
+page
+  h1.is-title-1 Cypress - Bypassing the UI
+  p This page is used exclusively by Cypress to perform repetitive actions. &nbsp;
+    a.link(href='https://docs.cypress.io/guides/getting-started/testing-your-app.html#Bypassing-your-UI' target='_blank') Read more about this strategy.
+  br
+  .card
+    h2.is-title-2 Performing Action &nbsp;
+      span.has-text-success(data-test='actionName') {{ ephemeral.action.name }}
+    div(v-if='ephemeral.action.name')
+      p.has-text-bold Queries:
+      p.c-query {{ JSON.stringify(ephemeral.action.queries) }}
+    div(v-else)
+      p.has-text-1 Nothing at the moment...
+
+    banner-scoped(ref='bannerAction')
+
+    button.is-success.c-btn(
+      v-if='ephemeral.isFinalized'
+      data-test='finalizeBtn'
+      @click='finalizeAction'
+    ) Finalize action
+
+  .card
+    h3.is-title-2 Actions available
+    p Pass the action name and its arguments as queries.
+    p.has-text-1 Ex: /bypass-ui?action=user_signup&username=john&email=john@email.com&password=123456789
+    ul.c-list
+      li(v-for='(obj, actionName) in actions') {{ actionName }}
+</template>
+
+<script>
+import sbp from '~/shared/sbp.js'
+import Page from '@pages/Page.vue'
+import BannerScoped from '@components/BannerScoped.vue'
+
+export default {
+  name: 'BypassUI',
+  components: {
+    Page,
+    BannerScoped
+  },
+  data () {
+    return {
+      ephemeral: {
+        action: {},
+        isFinalized: false
+      }
+    }
+  },
+  async mounted () {
+    const { action, ...queries } = this.$route.query
+
+    if (!action) {
+      return false
+    }
+
+    const { actionFn } = this.actions[action] || {}
+
+    if (!actionFn) {
+      // Wait for $refs.bannerAction to exist
+      this.$nextTick(() => {
+        this.$refs.bannerAction.danger(`Action ${action} doesn't exist.`)
+      })
```

Comment:
If it doesn't exist then we shouldn't proceed below to call it

Reply:
That's why we use $nextTick. To make sure the banner is already mounted.

Reply:
Sorry, I wasn't clear, I meant if `actionFn` doesn't exist then we should `return` here and not proceed to line 67.